### PR TITLE
docs(collapse): add demo with show/hide methods

### LINKED ISSUE #3591: docs(collapse): add manual show/hide demo
adding manual demo with show/hide methods(#3238)

### PR DESCRIPTION
**Pull request by tarusin**
close #3591

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+collapse/demos/toggle-manual/toggle-manual.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<button type="button" class="btn btn-primary"
+        (click)="hideContent()">Show content
+</button>
+<button type="button" class="btn btn-primary"
+        (click)="showContent()">Hide content
+</button>
+<hr>
+<div *ngIf="showSelected"
```
Comment (IlyaSurmay) [inline demo/src/app/components/+collapse/demos/toggle-manual/toggle-manual.component.html]: Why do we need a basic demo with `*ngIf` that doesn't even use CollapseDirective?

File: demo/src/app/components/+collapse/collapse-section.list.ts [hunk 2]
```diff
@@ -24,9 +25,16 @@ export const demoComponentContent: ContentSection[] = [
       {
         title: 'Basic',
         anchor: 'basic',
-        component: require('!!raw-loader?lang=typescript!./demos/collapse-demo.component'),
-        html: require('!!raw-loader?lang=markup!./demos/collapse-demo.component.html'),
+        component: require('!!raw-loader?lang=typescript!./demos/basic/collapse-demo.component'),
```
Comment (IlyaSurmay) [inline demo/src/app/components/+collapse/collapse-section.list.ts]: ts and html files in folder should have the same names

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+collapse/demos/toggle-manual/toggle-manual.component.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'toggle-manual-demo',
+  templateUrl: './toggle-manual.component.html'
+})
+export class ToggleManualDemoComponent {
+  showSelected = true;
+  hideContent() {
+    this.showSelected = true;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+collapse/demos/toggle-manual/toggle-manual.component.ts]: show / hide must be done without ngIf, using `hide()` and `show()` methods of `CollapseDirective`

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**